_pages/about.md

```diff
@@ -60,22 +60,22 @@ flow:
             **Collaborate on common software projects with Linaro and other industry leaders**
 
 
-            Become a Linaro member and work with Linaro, other industry leaders and the open source community on software projects which deliver standardization and help accelerate new technologies on Arm. There are three levels of membership you can choose from - Core, Club and Group (Linaro has four Segment Groups - Linaro Consumer Group, Linaro Datacenter & Cloud Group, Linaro Edge & Fog Computing Group and Linaro IoT and Embedded Group). For more information on Linaro membership, how to participate in projects and what value our members see in membership, go to [www.linaro.org/membership](/membership/). 
+            Become a Linaro member and work with Linaro, other industry leaders and the open source community on software projects which deliver standardization and help accelerate new technologies on Arm. There are three levels of membership you can choose from - Core, Club and Group (Linaro has four Segment Groups - Linaro Consumer Group, Linaro Datacenter & Cloud Group, Linaro Edge & Fog Computing Group and Linaro IoT and Embedded Group). For more information on Linaro membership, how to participate in projects and what value our members see in membership, go to [Linaro Memberships](https://www.linaro.org/membership/)[](/membership/). 
 
 
             **Leverage Linaro’s software expertise on your project**
 
 
-            If you would rather work with Linaro on a one-to-one basis and are interested in leveraging Linaro’s Arm software expertise on your project, go to [www.linaro.org/services](/services/) to speak to Linaro Developer Services. Working with our Services team allows you to leverage Linaro’s Arm software expertise and experience working with, and as part of, open source communities. 
+            If you would rather work with Linaro on a one-to-one basis and are interested in leveraging Linaro’s Arm software expertise on your project, go to [Linaro Services](https://www.linaro.org/services/). Working with our Services team allows you to leverage Linaro’s Arm software expertise and experience working with, and as part of, open source communities. 
```

Comment:
The URL should have been left as `(/services/)`.

Reply:
Sorry can you remind me why I should leave it as /services/? Obviously I was doing this fairly late and forgot but do remember you mentioning this prevoiusly. Apologies.